core/i18n/package.json

```diff
@@ -43,28 +46,33 @@
     "@itwin/core-bentley": "workspace:*",
     "@itwin/core-common": "workspace:*",
     "@itwin/eslint-plugin": "workspace:*",
+    "@itwin/certa": "workspace:*",
     "@types/node": "16.11.7",
+    "@types/mocha": "^8.2.2",
+    "@types/chai": "4.3.1",
     "@types/i18next": "^8.4.2",
     "@types/i18next-browser-languagedetector": "^2.0.1",
     "eslint": "^7.11.0",
     "rimraf": "^3.0.2",
-    "typescript": "~4.4.0"
+    "typescript": "~4.4.0",
+    "webpack": "^5.64.4",
+    "source-map-loader": "^4.0.0",
+    "chai": "^4.1.2"
   },
   "//dependencies": [
     "NOTE: these dependencies are specific to core-i18n",
     "NOTE: these dependencies should be only for things that DO NOT APPEAR IN THE API",
     "NOTE: core-i18n should remain UI technology agnostic, so no react/angular dependencies are allowed"
   ],
   "dependencies": {
-    "i18next": "^21.2.6",
+    "i18next": "21.9.0",
```

Comment:
Why are we locking the version?

Reply:
i18next changed a variable from a `string` to a `string | false | readonly string[]` which was tricky to deal with nicely, so I had locked it for the time being and was going to attempt to update it when finishing the i18n tests. But I just gave it another try and figured it out, so all good now.

Reply:
cool, sounds good.